cli/parser/parser.go

```diff
@@ -118,6 +121,11 @@ func (m *Subparser) Add(d *options.Definition) error {
 	if _, ok := m.ByShortName[d.ShortName()]; ok {
 		return fmt.Errorf("Naming collision adding flag with short name %s; flag already exists with that short name.", d.ShortName())
 	}
+	if d.HasNegative() {
+		if _, ok := m.ByName["no"+d.Name()]; ok {
```

Comment:
darn, I guess this means we can't have a string flag called `norway` and boolean flag called `rway` :P